server/src/main/java/org/candlepin/model/OwnerCurator.java

```diff
@@ -78,6 +78,13 @@ public Owner findOwnerById(String ownerId) {
         return owner;
     }
 
+    public boolean ownerLoggingEnabled() {
+        Long ownersWithLogging = (Long) getEntityManager()
+            .createQuery("SELECT COUNT(o) FROM Owner o WHERE logLevel IS NOT NULL")
```

Comment:
This is likely to fall over in production given the number of owners that are going to be scanned.

Reply:
I think it should still be pretty darn fast:

```
MariaDB [candlepin]> select count(*) from cp_owner where loglevel is not null;
+----------+
| count(*) |
+----------+
|        5 |
+----------+
1 row in set (0.29 sec)

MariaDB [candlepin]> select count(*) from cp_owner;
+----------+
| count(*) |
+----------+
|  1188919 |
+----------+
1 row in set (0.23 sec)
```

That's from a production dump from August.  So scanning 1.2 million owners takes about a quarter of a second which they should make back up within probably under an hour of not having the filter running.

Reply:
fair enough